app/src/main/java/org/openedx/app/AppRouter.kt

```diff
@@ -113,13 +112,9 @@ class AppRouter : AuthRouter, DiscoveryRouter, DashboardRouter, CourseRouter, Di
     override fun navigateToNoAccess(
         fm: FragmentManager,
         title: String,
-        coursewareAccess: CoursewareAccess,
         auditAccessExpires: Date?
     ) {
-        replaceFragment(
-            fm,
-            NoAccessCourseContainerFragment.newInstance(title, coursewareAccess, auditAccessExpires)
-        )
+        replaceFragment(fm, NoAccessCourseContainerFragment.newInstance(title, auditAccessExpires))
```

Comment:
IMO, we can remove the `auditAccessExpires` parameter as well

Reply:
It looks like the `NoAccessCourseContainerFragment` uses this parameter.

<img width="614" alt="image" src="https://github.com/openedx/openedx-app-android/assets/127732735/31cc664b-7471-4e7e-b7ab-d1cbe06b7378">

Reply:
[Router.NavigateToNoAccess](https://github.com/openedx/openedx-app-android/pull/171/files/b26610295e36760ffb1b4cc81ac2c22519283cdc#diff-afd03945c123e5bcb0cdc134a628d8ea040dbb3bf0a7567c133261d6d3bf4151R112) is always getting null for `auditAccessExpires`

Reply:
Got it, I'll check it

Reply:
Yeah, you're right. I found the `null` value and removed the `auditAccessExpires` argument